scripts/run-e2e.ts

```diff
@@ -196,8 +181,6 @@ const runTests = async ({ name, ...rest }: Parameters) => {
   try {
     if (shouldUseTestRunner) {
```

Comment:
Shouldn't this always be `true` now?

Reply:
Looks like with this change it will always be `false` and the test runner will never run

Reply:
@shilman please help me understand what did I change that impacts the testrunner?

this value is coming from a cli-flag?
https://github.com/storybookjs/storybook/blob/497837346f01e320c3e7cb6d42d9f2e56b0f7101/scripts/run-e2e.ts#L61

Reply:
@ndelangen You removed the flag https://github.com/storybookjs/storybook/pull/19303/files#diff-dba89a5f00386a7c76cb45dcd9dbc8ec6c9027b20c0b371435fc867349bbf840L44

Reply:
Oops that was not intentional!

Reply:
Should remove the if statement instead (and remove the cli flag).